### PR TITLE
start building python graphical tests

### PR DESCRIPTION
**Pull request by sanuj**
For #3046.
I'm in the process of learning CMake. Please review.

### CODE REVIEW

File: examples/undocumented/python_modular/graphical/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+INCLUDE(PythonEnvironment)
+GET_PYTHON_ENV()
+
+# FILE(GLOB PYTHON_EXAMPLES *.py)
```
Comment (sanuj) [inline examples/undocumented/python_modular/graphical/CMakeLists.txt]: We can uncomment this to build all the graphical examples but work needs to be done on the graphical examples to make them build without error. Currently only `classifier_gaussian_process_binary_classification.py` is tested.

**Comment by sanuj** [comment]
[Dunno why it's failing.](https://travis-ci.org/shogun-toolbox/shogun/jobs/120876406#L3143) Passes on my local.

**Comment by karlnapf** [comment]
As said, check the travis log, search for "error" and you will see that matplotlib is not there

File: examples/undocumented/python_modular/graphical/classifier_gaussian_process_binary_classification.py [hunk 2]
```diff
@@ -7,25 +7,27 @@
 # Written (W) 2013 Roman Votyakov
 #
 
-from pylab import *
-from numpy import *
-from itertools import *
+import matplotlib
+matplotlib.use('Agg')   # Comment this to see the plot
```
Comment (karlnapf) [inline examples/undocumented/python_modular/graphical/classifier_gaussian_process_binary_classification.py]: this should not be in the code file. It should only be put there when the script is called from the cmake test

**Comment by karlnapf** [comment]
I close this for now....Feel free to re-open if you wanna continue, but it is not really there yet

**Pull request closed without merging**